PEA/Makefile

```diff
@@ -11,7 +11,9 @@ empty :=
 space := $(empty) $(empty)
 # CTWRunner accepts arguments with commas instead of spaces
 CTW_RUNNER_JAVA_OPTIONS := $(subst $(space),$(comma),$(CTW_JAVA_OPTIONS))
-JTREG_COMMON := CONF=linux-x86_64-server-fastdebug  JTREG="JAVA_OPTIONS=-XX:+UnlockExperimentalVMOptions -XX:+DoPartialEscapeAnalysis"
+CONF ?= linux-x86_64-server-fastdebug
+JTREG_COMMON = JAVA_OPTIONS=-XX:+UnlockExperimentalVMOptions -XX:+DoPartialEscapeAnalysis
+JTREG_PARANOID=VM_OPTIONS=-ea -esa -XX:CompileThreshold=100
```

Comment:
should we add `-XX:PEAParanoid`?

Reply:
we will, but it still have many failure so far.